davealex/quick-credit
# #165953647 Test and Implement sign in API

#### What does this PR do?
Test and Implement signup API endpoint
#### Description of Task to be completed?
Write Unit tests for API v1 endpoint: `/auth/signin`
Implement endpoint and validate requests
#### How should this be manually tested?
Using Postman, make a request  to the API endpoint POST `/api/v1/signin` and ensure that the appropriate JSON responses are returned (when passing validation, and when failing validation)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165953647](https://www.pivotaltracker.com/story/show/165953647)
#### Screenshots (if appropriate)
#### Questions:
N/A
